src/utils/versionVerification.ts

```diff
@@ -0,0 +1,27 @@
+import fetch from 'node-fetch';
+import { promises as fs } from 'fs';
+
+export async function getLatestRelease(owner: string, repo: string) {
+  const response = await fetch(
+    `https://api.github.com/repos/${owner}/${repo}/releases/latest`,
+  );
+  const data = await response.json();
+  return data.tag_name;
+}
+
+export async function checkLatestVersion() {
+  const owner = 'asyncapi';
+  const repo = 'cli';
+
+  const latestVersion:string = await getLatestRelease(owner, repo);
+  // Read the package.json file to get the current version
+  const packageJson = JSON.parse(await fs.readFile('package.json', 'utf-8') as string);
+  const currentVersion = `v${packageJson.version}`;
+  // Compare the versions
+  if (latestVersion !== currentVersion) {
+    // Avoid sending message frequently
+    const randomProbability = Math.random();
+    if (randomProbability < 0.5) {return `There is a newer version available: ${latestVersion}. You are currently using ${currentVersion}.`;}
+  }
+}
+checkLatestVersion();
```

Comment:
Is there any reason the function is being called here?

Reply:
thanks @Shurtu-gal no used, I was testing some changes removed

Reply:
@Shurtu-gal If we are doing infrequent messages, how can we test this 🤔